### DISCUSSION TITLE
How to apply AURA CREATOR effects on leeboard lighting

### DISCUSSION THREAD

**Discussion by ma7876** [seerge/g-helper#1770] [Q&A] [answered]
I have exported my Aura Creator effects and I want to apply them to my keyboard, but I cant do it in Aura Creator as it gives me an error, saying that there are service problems.

**Comment by seerge** [comment]
This is something you should be asking from Asus support, not here :)

**Reply by ma7876** [reply]
No, I mean I deleted Armory create fully to use only g-helper, but in g-helper there is no such an option to apply custom effects from aura creator right? So I was forced to install Armory create to make aura creator work to apply custom effect, here:
![Screenshot 2023-12-21 190819](https://github.com/seerge/g-helper/assets/146047099/e14099a5-10e2-4a62-87fc-26682e443dff)

So should I have Armory create installed to use custom lighting effects or there is another way to apply it in g-helper?

**Reply by seerge** [reply, marked as answer]
Aura Creator is asus software :) it has nothing to do with g-helper. G-Helper doesn't interact with asus apps anyhow.

So, most probably, you will need Armoury with it's services to keep Aura creator working

**Reply by ma7876** [reply]
I accidentally made your reply an answer sorry, the thing is g-helper is supped to be replacement for Armory create right? What I am saying is aura creator does not work without Armory create, it gives me this error:
![Screenshot 2023-12-21 191444](https://github.com/seerge/g-helper/assets/146047099/a94ac411-0062-4bfa-8a27-9ec8d429e3fd)

So it means just using only g-help I am not able to use aura creator, as it needs Armory create to work, what I am suggesting maybe you can integrate functions to g-helper to it to provide needed services for Aura creator for it to work without Armory create

**Reply by seerge** [reply]
@ma7876 I'm not sure if you have read what I wrote. Aura Creator is part of Armoury Crate package, it depends on it's services.

G-Helper is a lightweight replacement, it's physically impossible to "integrate" 2GB of someone-else's proprietary software here anyhow :)
